sdk/cosmos/test-resources.bicep

```diff
@@ -0,0 +1,66 @@
+param baseName string
+
+@description('Flag to enable or disable multiple write locations on CosmosDB Account')
+param enableMultipleWriteLocations bool
+
+@description('Default Cosmosdb Account level consistency')
+param defaultConsistencyLevel string = 'Session'
+
+@description('Enable multiple regions, default value is false')
+param enableMultipleRegions bool = false
+
+@description('Location for the Cosmos DB account.')
+param location string = resourceGroup().location
+
+@description('The api version to be used by Bicep to create resources')
+param apiVersion string = '2020-04-01'
+
+var accountName = toLower(baseName)
+var resourceId = cosmosAccount.id
+var singleRegionConfiguration = [
+  {
+    locationName: 'East US 2'
```

Comment:
I'm not sure yet how we'll solve this for multi-region, but for single region you could also set the location name to `resourceGroup().location` (unless it gives the short name, if that won't work).

Also one other thing to consider is hardcoding regions will break the tests if/when we try to add sovereign cloud support. We'll have to figure this out anyway in more detail for multi-region though.

Reply:
Also, which region configuration will be used in the tests? Single? Is the idea that locally you may do multi-region, or vice versa?

Would we want to support testing BOTH configurations in the pipeline?

Reply:
@v-xuto has experience converting tests to work with sovereign cloud. Here's an example PR for Search: https://github.com/Azure/azure-sdk-for-python/pull/23005

Reply:
@benbp we do seem to use location for the `cosmosAccount` so it can probably be used for the single region - these settings are based off the Java SDK's own configs for test-resources.json and are supposed to be used in our multimaster tests, but not sure exactly how this gets set since this deployment only runs once right?